src/libmad/huffman.c

```diff
@@ -51,7 +51,7 @@
 # endif
 
 static
-const union huffquad const hufftabA[] PROGMEM = {
```

Comment:
What's the problem GCC has with `static const`?  Seems like the PROGMEM would require it anyway, no?

Reply:
Presence of two `const` is annoying gcc.